jib-core/src/main/java/com/google/cloud/tools/jib/image/Image.java

```diff
@@ -138,8 +141,10 @@
      * @param volumes the list of directories to create a volume.
      * @return this
      */
-    public Builder<T> setVolumes(@Nullable Set<AbsoluteUnixPath> volumes) {
-      this.volumes = (volumes == null) ? null : ImmutableSet.copyOf(volumes);
+    public Builder<T> addVolumes(@Nullable Set<AbsoluteUnixPath> volumes) {
```

Comment:
Maybe update the javadoc, since these two methods are not setters anymore.

Reply:
Good point